### PR TITLE
FEAT: add CheckSwitchLanguage in Test.cs, create LanguagesPage.cs

### PR DESCRIPTION
**Pull request by khayitmuradov**
created LanguagesPage.cs in Pages folder
added CheckSwitchLanguage() method in Test.cs

### CODE REVIEW

File: UI/Pages/LanguagesPage.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{
+    public class LanguagesPage : AbstractPage
+    {
+        private static readonly By languagesPopUpWindow = By.Id("p-lang-btn");
+        private static readonly By russianLanguageLink = By.LinkText("Русский");
+        private static readonly By englishLanguageLink = By.CssSelector(".interwiki-en span");
+        private static readonly By uzbekLanguagePage = By.LinkText("Uzbek Wikipedia");
+        private static readonly By uzbekWikiPedia = By.LinkText("uz.wikipedia.org");
+
+        public void ClickPopUpWindowButton()
```
Comment (NikitaGavruk) [inline UI/Pages/LanguagesPage.cs]: Make it please via fluent page object pattern

File: UI/Tests/Test.cs [hunk 2]
```diff
@@ -73,5 +73,23 @@ public void CheckNotLoggedWarningOnEditPage()
...
+            var languagesPage = new LanguagesPage();
+            Driver.Navigate().GoToUrl("https://en.wikipedia.org/wiki/List_of_Wikipedias");
+            string expectedTitle = "Vikipediya";
+            CustomLogger.LogInfo(LogLevel.Info, "Go to Languages Page");
+            languagesPage.ClickPopUpWindowButton();
+            languagesPage.ClickRussianLanguageButton();
+            Assert.That(Driver.Title, Is.EqualTo("Языковые разделы Википедии — Википедия"));
+            languagesPage.ClickEnglishLanguageButton();
+            Assert.That(Driver.Title, Is.EqualTo("List of Wikipedias - Wikipedia"));
+            languagesPage.ClickUzbekLanguageButton();
```
Comment (NikitaGavruk) [inline UI/Tests/Test.cs]: Add please log info here too as you did it on 85 line

File: UI/Tests/Test.cs [hunk 3]
```diff
@@ -73,5 +73,23 @@ public void CheckNotLoggedWarningOnEditPage()
...
+            languagesPage.ClickUzbekWikipediaButton();
```
Comment (NikitaGavruk) [inline UI/Tests/Test.cs]: Why you did not verify Uzbek language?

**Review by NikitaGavruk — CHANGES_REQUESTED** [review]

**Comment by NikitaGavruk** [comment]
What does it mean "FEAT" at name of this PR?

**Comment by khayitmuradov** [comment]
FEAT means feature no?  I read it from that documentation about commiting which you gave

**Comment by khayitmuradov** [comment]
> What does it mean "FEAT" at name of this PR?
Reworked Languages Part: #21

**Comment by NikitaGavruk** [comment]
:) I did not read it. Anyway we do not need it. We just need to have clear commits name

**Review by NikitaGavruk — APPROVED** [review]

**Pull request merged**